app/models/user.rb

```diff
@@ -11,10 +9,43 @@ class User < ActiveRecord::Base
          :trackable,
          :validatable
 
+  after_create :send_admin_mail
+
   belongs_to :agency
 
-  scope :case_workers_by_agency, ->(agency = nil) { case_workers.where(agency: agency) }
-  scope :case_workers, -> { where(role: :case_worker) }
+  def self.case_workers_by_agency(agency = nil)
```

Comment:
Just returning code back to it's original implementation.  I wasn't clear why the class methods I had before got transformed into scopes.  I feel class methods read better and makes it feel more like a regular ruby object.  I find scopes start to add more complexity when queries start getting complicated.  For these queries it's just a personal preference and I lean toward simple ruby class methods.